public/locales/fr/settings.json

```
@@ -0,0 +1,8 @@
+{
+  "settingsFor": "Réglages pour",
+  "saveSettings": "Sauvegarder les réglages",
```

May we make this one shorter so it fits better in the button? It may be problematic on mobile.
```suggestion
  "saveSettings": "Sauvegarder",
```